polaris-react/src/components/AlphaFilters/AlphaFilters.tsx

```diff
@@ -360,7 +360,7 @@ export function AlphaFilters({
           </div>
         </div>
         {hideQueryField ? (
-          <Box paddingInlineEnd="2" paddingBlockStart="2">
+          <Box paddingInlineEnd="2" paddingBlockStart="2" paddingBlockEnd="2">
```

Comment:
Tophatted in storybook and just spotted difference in right padding when toggling display filters:

https://user-images.githubusercontent.com/701295/235190504-103d59fb-1b27-4993-9dfa-0f9f84a04fa9.mov

```suggestion
          <Box paddingInlineEnd="3" paddingBlockStart="2" paddingBlockEnd="2">
```

Reply:
Good catch! Thanks, @alllx!